docs/dev/testservers.rst

```diff
@@ -3,48 +3,164 @@
+|                                            | - 3 Nodes                   |                |                     |                |
+|                                            | - Iris                      |                |                     |                |
+|                                            | - LTI                       |                |                     |                |
++--------------------------------------------+-----------------------------+----------------+---------------------+----------------+
+| https://artemis-test4.artemis.cit.tum.de   | - Integrated Code Lifecycle |     GitHub     | - TUMonline account |    On Demand   |
+|                                            | - LDAP                      |                | - Test accounts     |                |
+|                                            | - LTI                       |                |                     |                |
+|                                            | - Apollon                   |                |                     |                |
+|                                            | - Iris                      |                |                     |                |
+|                                            | - MySQL                     |                |                     |                |
++--------------------------------------------+-----------------------------+----------------+---------------------+----------------+
+| https://artemis-test5.artemis.cit.tum.de   | - Integrated Code Lifecycle |     GitHub     | - TUMonline account |    On Demand   |
+|                                            | - Postgres                  |                | - Test accounts     |                |
+|                                            | - 1 Node                    |                |                     |                |
+|                                            | - Iris                      |                |                     |                |
+|                                            | - LTI                       |                |                     |                |
++--------------------------------------------+-----------------------------+----------------+---------------------+----------------+
+| https://artemis-test6.artemis.cit.tum.de   | - GitlabCI                  |     GitHub     | - TUMonline account |    On Demand   |
+|                                            | - Postgres                  |                | - Test accounts     |                |
+|                                            | - 1 Node                    |                |                     |                |
+|                                            | - LTI                       |                |                     |                |
++--------------------------------------------+-----------------------------+----------------+---------------------+----------------+
```

Comment:
What about TS9 and ma-schwind?

Reply:
ma schwind is tagged as a legacy server which is added now to link to confluence

Reply:
@bensofficial i dont know how ts9 is structured so maybe u can add it